python/doc/examples/meta_modeling/kriging_metamodel/plot_kriging.py

```diff
@@ -4,17 +4,17 @@
 """
 # %%
 # In this example we are going to create an approximation of a model response using a kriging model.
-# We consider a bidimensional function with gaussian inputs.
-# Then we create a kriging metamodel with a constant basis and a `SquaredExponential` covariance.
+# We consider a bidimensional function with Gaussian inputs.
+# Then we create a kriging metamodel with a constant basis and a :class:`~openturns.SquaredExponential` covariance.
 #
 # We consider the function
 #
 # .. math::
-#    g(X) = \cos(X_1 + X_2)
+#    g(\vect{X}) = \cos(X_1 + X_2)
```

Comment:
This is a deterministic function. Therefore, there is no reason to introduce a random vector at this level.
```suggestion
#    g(\vect{x}) = \cos(x_1 + x_2)
```